Src/Couchbase.Linq/BucketContext.cs

```diff
@@ -42,8 +46,8 @@ public ClientConfiguration Configuration
         }
 
         /// <summary>
-        /// Queries the current <see cref="IBucket" /> for entities of type <see cref="T" />. This is the target of
-        /// the Linq query requires that the associated JSON document have a type property that is the same as <see cref="T" />.
+        /// Queries the current <see cref="IBucket" /> for entities of type T. This is the target of
+        /// the Linq query requires that the associated JSON document have a type property that is the same as T.
```

Comment:
should be "This is the target of a Linq query and requires that the associated JSON document have a type property that is the same as..."